components/switch/script.js

```diff
@@ -6,12 +6,18 @@
 import components from 'coherent-gameface-components';
 import textInside from './templates/text-inside-template.html';
 import textOutside from './templates/text-outside-template.html';
-
+const CustomElementValidator = components.CustomElementValidator;
 /**
  * @class Switch
  * Switch component, that allows you to switch between true and false
  */
-class Switch extends HTMLElement {
+class Switch extends CustomElementValidator {
+    get value() {
+        const value = this.getAttribute('value');
+        if (this.isFormElement(this)) return value || 'on';
```

Comment:
The[ standard behavior of the checkbox is to return 'on'](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#attr-value) if there is no value. There is no standard switch, but essentially it is a checkbox, so we apply the same rules.
[The empty string is a falsy value](https://developer.mozilla.org/en-US/docs/Glossary/Falsy), so in the logical OR operator above it will return 'on'.

Reply:
If checkbox has value attribute that is empty string it will be included to the submitted url and it won't be 'on'.
For example submitting
```
<input type="checkbox" value="" name="gosho" checked>
<input type="checkbox" value="alalbal" name="gosho2" checked>
```
will result to `gosho=&gosho2=alalbal`